cit3.0-web/src/components/Map/Map.js

```diff
@@ -133,7 +130,7 @@ export default function Map({
           : null}
       </>
     );
-    zoomLevel = 2;
+    zoomLevel = 5; // do not change!
```

Comment:
How come this is this kept at 5?

Reply:
5 gives us an initial view of all of BC, 2 gives us all of North and South America :-)